assets/js/base/components/cart-checkout/totals/shipping/shipping-rate-selector.tsx

```diff
@@ -38,6 +45,7 @@ const ShippingRateSelector = ( {
 						) }
 					</Notice>
 				}
+				renderOption={ renderPackageRateOption }
```

Comment:
Instead of passing `renderPackageRateOption` here please check where it actually gets used. You'll see that in https://github.com/woocommerce/woocommerce-blocks/blob/df63d2ae397b35242c40ea4eec4673c50eaabb15/assets/js/base/components/cart-checkout/shipping-rates-control-package/package-rates.tsx#L19 it gets a default value, which means we can leave it undefined here. We'd just have to fix the types in `ShippingRatesControl` , and any further components that use it to allow this prop to also be undefined.

Reply:
Yes, I struggle a lot to fix the type of renderOption. It didn't cross my mind to make it undefined to match the type. Thank you for the suggestion. I have made the changes. 👍